### DISCUSSION TITLE
Netcoredbg is opening new console instead of attaching to dap-ui's

### DISCUSSION THREAD

**Discussion by Wiebesiek** [mfussenegger/nvim-dap#933] [General]
Is it possible to have Netcoredbg attach to an integrated terminal? It looks as though netcoredbg supports it. https://github.com/mfussenegger/nvim-dap/issues/763 I found a similar issue here with Go lang.

**Comment by Wiebesiek** [comment]
I should note, everything else seems to be working fine with dap and dap-ui. It's just that it spawns a new terminal. From the dap log I can see what looks like a request for an integrated terminal with `supportsRunInTerminalRequest = true`. 
```
[ DEBUG ] 2023-04-23T14:05:11Z-0700 ] ...data\site\pack\packer\start\nvim-dap/lua/dap/session.lua:1588 ]	"request"	{
  arguments = {
    adapterID = "nvim-dap",
    clientId = "neovim",
    clientname = "neovim",
    columnsStartAt1 = true,
    linesStartAt1 = true,
    locale = "en_US",
    pathFormat = "path",
    supportsProgressReporting = true,
    supportsRunInTerminalRequest = true,
    supportsStartDebuggingRequest = true,
    supportsVariableType = true
  },
  command = "initialize",
  seq = 0,
  type = "request"
}
```

**Comment by redfoggg** [comment]
You ever came to a conclusion in this?? I'm facing similar issue with C#, my problem is that everything goes to REPL instead of the integratedTerminal, then I'm not capable to debug console applications with C# and nvim-dap with this plugin configuration.

**Reply by Wiebesiek** [reply]
https://github.com/Wiebesiek/ZeoVim
The fundamental issue exists still for me, but I was able to make it usable by putting it in a `detached` state. https://github.com/Wiebesiek/ZeoVim/blob/5a291c4da40f4880c313538459195067b6abb475/lua/zeovim/config/dap.lua#L25C29-L25C29
